### DISCUSSION TITLE
Can the spacing in the image be controlled?

### DISCUSSION THREAD

**Discussion by 262146150** [opensheetmusicdisplay/opensheetmusicdisplay#1576] [Q&A] [answered]
![WX20241209-103110@2x](https://github.com/user-attachments/assets/97ac23c9-a875-40a0-b386-ab272edac014)

Can I control the spacing between the red arrows? I might add some drawings above certain notes, such as indicating that this is the beginning of a phrase, and there will be multiple such drawings.

**Comment by sschmidTU** [comment, marked as answer]
Yes, for example, this is triple the spacing (default is 7):
```js
osmd.EngravingRules.MinimumDistanceBetweenSystems = 21;
// re-render
```
